applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -4,3 +4,173 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#if (CONFIG_AUDIO_DEV == GATEWAY)
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/audio/audio.h>
+
+#include "le_audio.h"
+#include "macros_common.h"
+#include "ctrl_events.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(bis_gateway, CONFIG_LOG_BLE_LEVEL);
+
+/* When BROADCAST_ENQUEUE_COUNT > 1 we can enqueue enough buffers to ensure that
+ * the controller is never idle
+ */
+#define BROADCAST_ENQUEUE_COUNT 2U
+#define TOTAL_BUF_NEEDED (BROADCAST_ENQUEUE_COUNT * CONFIG_BT_AUDIO_BROADCAST_SRC_STREAM_COUNT)
+
+BUILD_ASSERT(CONFIG_BT_ISO_TX_BUF_COUNT >= TOTAL_BUF_NEEDED,
+	     "CONFIG_BT_ISO_TX_BUF_COUNT should be at least "
+	     "BROADCAST_ENQUEUE_COUNT * CONFIG_BT_AUDIO_BROADCAST_SRC_STREAM_COUNT");
+
+static struct bt_audio_lc3_preset lc3_preset = BT_AUDIO_LC3_BROADCAST_PRESET_48_4_1;
+// TODO: Currently only 1 stream is supported
+static struct bt_audio_stream streams[CONFIG_BT_AUDIO_BROADCAST_SRC_STREAM_COUNT];
```

Comment:
Is this because of an issue in the sample? Or could we in theory here start two streams and send left/right to each its stream?

Reply:
It should be possible to start more than one stream when we have implemented the required functionality in these le_audio_bis_gateway/headset files. I have not looked at that yet.